dateutil/tz/tz.py

```diff
@@ -1456,15 +1456,37 @@ def __repr__(self):
         return "%s(%s)" % (self.__class__.__name__, repr(self._s))
 
 
-if sys.platform != "win32":
-    TZFILES = ["/etc/localtime", "localtime"]
-    TZPATHS = ["/usr/share/zoneinfo",
-               "/usr/lib/zoneinfo",
-               "/usr/share/lib/zoneinfo",
-               "/etc/zoneinfo"]
-else:
-    TZFILES = []
-    TZPATHS = []
+def __get_tz_files():
+    env_value = os.environ.get('DATEUTIL_TZFILES', None)
+    if env_value is not None:
+        return env_value.split(':')
+
+    if sys.platform == "win32":
+        return []
+
+    return ["/etc/localtime", "localtime"]
+
+
+def __get_tz_paths():
+    env_value = os.environ.get('DATEUTIL_TZPATH', None)
```

Comment:
@pganssle what do you think we should do if an empty env var is set?

I was going to consider empty as not set, but I guess this can be used as a way to force dateutil to pick up the vendored tzdata information.

Reply:
I think most people would want only the first one, at least in my experience.

If I am an end-user I will know where to look at and just set it. If I am a framework developer or the like I will probably set it as a default and allow users to reset it if they want.

In short: I'll change the  PR to make `''` be equal to not set. For `DATEUTIL_DEFAULT_TZPATH` I think YAGNI.